src/Authentication/Authentication/Cmdlets/GetMgGraphOption.cs

```diff
@@ -0,0 +1,36 @@
+// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+using Newtonsoft.Json.Linq;
+using System.IO;
+using System.Management.Automation;
+
+namespace Microsoft.Graph.PowerShell.Authentication.Cmdlets
+{
+    [Cmdlet(VerbsCommon.Get, "MgGraphOption", HelpUri = "")]
+    public class GetMgGraphOption : PSCmdlet
+    {
+        protected override void BeginProcessing()
+        {
+            base.BeginProcessing();
+        }
+
+        protected override void ProcessRecord()
+        {
+            base.ProcessRecord();
+            //File.WriteAllText(Constants.GraphOptionsFilePath, JsonConvert.SerializeObject(GraphSession.Instance.GraphOption, Formatting.Indented));
+            WriteObject(new PSObject(JObject.Parse(File.ReadAllText(Constants.GraphOptionsFilePath))));
```

Comment:
You should be able to use `GraphOption` directly from the session object. The session bound `GraphOption` is populated on module import from the file and updated via `Set-MgGraphOption`.
```suggestion
            WriteObject(GraphSession.Instance.GraphOption);
```